src/main/java/bowling_step3/domain/BowlingGame.java

```diff
@@ -0,0 +1,76 @@
+package bowling_step3.domain;
+
+import bowling_step3.exception.PitchOverBoundException;
+
+import java.util.Objects;
+
+import static java.util.Collections.unmodifiableList;
+
+public class BowlingGame {
+
+    private final Player player;
+
+    private final GameFrames gameFrames;
+
+    public BowlingGame(Player playerName) {
+        this.player = playerName;
+        this.gameFrames = new GameFrames();
+    }
+
+    public void pitch(int countOfKnockDown) {
+        if (isFinish()) {
+            throw new PitchOverBoundException();
+        }
+
+        gameFrames.pitch(countOfKnockDown);
+    }
+
+    public boolean isFinish() {
+        return gameFrames.isFinish();
+    }
+
+    public GameFrames getGameFrames() {
+        return this.gameFrames;
+    }
+
+    private Scores gameScores() {
+        return gameFrames.getScores();
+    }
+
+    private int sumScore(int sum, int score) {
+        return sum + score;
+    }
+
+    public Scores addScores() {
+        Scores scores = Scores.of();
+
+        int sum = 0;
+        for (Integer score : unmodifiableList(gameScores().getScores())) {
+            sum = sumScore(sum, score);
+            scores.add(sum);
+        }
+
+        return scores;
+    }
```

Comment:
```suggestion
    public Scores sumScores() {
        Scores scores = Scores.of();

        int sum = 0;
        for (int score : gameFrames.getScores().getScores()) {
            sum += sum;
            scores.add(sum);
        }

        return scores;
    }
```
이렇게 표현할 수 있을 것 같습니다.
혹은 이 부분도 Scores에게 위임하여
```suggestion
    public Scores sumScores() {
        return Scores.sumOf(gameFrames.getScores().getScores());
    }
```
이렇게 표현할 수 있을 것 같아요!